src/components/steps/general-info/Funders.tsx

```diff
@@ -87,7 +93,7 @@ const Funders: React.FC<FundersProps> = ({ project, onChange }) => {
                     }}
                     onSave={() => {
                         const newAdditional = _(project.funders)
-                            .map(funder => funder.code)
+                            .map(funder => funder.code?.split("_").pop())
```

Comment:
[no need to change] Note that `xs.pop()` mutates `xs`. Here it's not a problem, because the array comes from an intermediate split, but as a rule of thumb prefer `_.last(xs)`